pylossless/pipeline.py

```diff
@@ -1037,13 +1037,19 @@ def save(self, derivatives_path, overwrite=False):
             path of the derivatives folder to save the file to.
         overwrite : bool (default False)
             whether to overwrite existing files with the same name.
+        format : str (default "EDF")
+            The format to use for saving the raw data. Can be 'auto',
+            'FIF', 'EDF', 'BrainVision', 'EEGLAB'.
+        event_id : dict | None (default None)
+            Dictionary mapping annotation descriptions to event codes.
         """
         mne_bids.write_raw_bids(
             self.raw,
             derivatives_path,
             overwrite=overwrite,
-            format="EDF",
+            format=format,
             allow_preload=True,
+            event_id=event_id,
```

Comment:
Per the mne-bids [documentation](https://mne.tools/mne-bids/stable/generated/mne_bids.write_raw_bids.html#mne_bids.write_raw_bids) of `write_raw_bids`: The `event_id` doc string says, "Descriptions or names describing the event codes, if you passed ``events``."

We don't pass anything to the `events` parameter of `write_raw_bids`. Can you confirm that passing `event_id` actually does anything in this case? If it does, maybe the `mne-bids` documentation is incorrect?